linkerd/docs/transformer.md

```diff
@@ -0,0 +1,41 @@
+# Transformer
+
+> Example Transformer Configuration
+
+```yaml
+routers:
+- ...
+  interpreter:
+    ...
+    transformers:
+    - kind: io.l5d.localhost
+```
+
+Transformers perform a transformation on the addresses resolved by the
+interpreter.  Transformations are applied sequentially in the order they appear.
+
+
+Key | Default Value | Description
+--- | ------------- | -----------
+kind | _required_ | One of the transformer kinds listed below.
+
+## Localhost
+
+kind: `io.l5d.localhost`
+
+The localhost transformer filters the list of addresses down to only addresses
+that have the same IP address as localhost.  This can be used by an incoming
+router to only route traffic to local destinations.
```

Comment:
I think we should try to explain the localhost-determination logic here.  (Resolve the local hostname in dns, non updating.)